#part 1 = goal, sent by client
uint32 start_demo
string bag_file_name
---
#part 2 = result, sent by server
#how many keyframes recorded
uint32 num_keyframes
---
#part 3 = feedback, sent periodically during execution
uint32 num_keyframes

